Tools/CISupport/ews-build/steps.py

```diff
@@ -915,6 +915,40 @@ def hideStepIf(self, results, step):
         return results == SUCCESS
 
 
+class InstallHooks(steps.ShellSequence):
+    name = 'install-hooks'
+    flunkOnFailure = False
+    haltOnFailure = False
+
+    def __init__(self, **kwargs):
+        super().__init__(timeout=30, logEnviron=False, **kwargs)
+
+    def run(self):
+        commands = [
+            ['git', 'config', 'include.path', '../metadata/git_config_extension'], [
```

Comment:
is ".." in "../metadata/git_config_extension" expected?

Reply:
Yes! It's got to be relative to `<repo>/.git`, and our `git_config_extension` is in `<repo>/metadata/git_config_extension`